kloppy/infra/serializers/event/opta/deserializer.py

```diff
@@ -378,15 +384,15 @@ def _parse_duel(
     raw_qualifiers: Dict[int, str], type_id: int, outcome: int
 ) -> Dict:
     qualifiers = _get_event_qualifiers(raw_qualifiers)
-    if type_id == EVENT_TYPE_TACKLE:
-        qualifiers.extend([DuelQualifier(value=DuelType.GROUND)])
-    elif type_id == EVENT_TYPE_AERIAL:
+    if type_id in [EVENT_TYPE_TACKLE, EVENT_TYPE_CHALLENGE]:
```

Comment:
For local code consistency, I'd use a tuple instead of a list.

`type_id in (EVENT_TYPE_TACKLE, EVENT_TYPE_CHALLENGE)`